File: exercicios_py/secao8/exercicio4.py
#Escreva algoritmo q leia e mostre um vetor de 20 elementos inteiros e mostre a soma dos elementos.

vetor = []
soma = 0

for i in range (0, 20):
    i = int(input("Digite um valor para o vetor: "))
    soma += i
    vetor.append(i)

print(soma)

File: exercicios_c/secao8/exercicio4.c
//Escreva algoritmo q leia e mostre um vetor de 20 elementos inteiros 
//e mostre a soma dos elementos.

#include<stdio.h>
int main(){
    int a[20], soma = 0;

    for(int i=0; i<20; i++){
        printf("Digite um valor: ");
        scanf("%d", &a[i]);
        soma += a[i];
    }printf("\nA soma dos elementos foi %d", soma);
}